eng/build.ps1

```diff
@@ -615,6 +614,7 @@ try {
   }
 
   if ($restore -or $build -or $rebuild -or $pack -or $sign -or $publish -or $testCoreClr) {
+    EnablePreviewSdks
```

Comment:
I'd only do this when restoring.

Reply:
Updated to only occur for CI builds plus local restore